spec/models/event_spec.rb

```diff
@@ -98,6 +98,23 @@
     end
   end
 
+  describe '#comments_count' do
+    context 'has a valid counter cache' do
+      before :each do
+        create(:comment, commentable_type: 'Event', commentable_id: event.id)
+      end
+
+      it 'successfully increments comments_count' do
+        expect(event.reload.comments_count).to eq 1
```

Comment:
I think the test should be more like:
`expect { create :comment, commentable: event }.to change { event.comments_count }.by(1)`